### DISCUSSION TITLE
scopeKey doesn't work?

### DISCUSSION THREAD

**Discussion by styopaiv** [wevm/wagmi#2031] [Q&A]
I have 3 instances of the same component which uses `useBalance({ address, scopeKey: "scope", watch: true })`, and i can see in the network tab that it fetches 3 times each new block (on top of `eth_blockNumber` request for each instance). How can we prevent spamming requests? 
Also is there a reason why there is no access to `refetchInterval` of react-query?

**Comment by iuriiiurevich** [comment]
@styopaiv the request deduplication issue was fixed in [1.0.9](https://github.com/wagmi-dev/wagmi/releases/tag/wagmi%401.0.9)
https://github.com/wagmi-dev/wagmi/issues/2442
